packages/data-api/scripts/createAnalyticsMaterializedView.sql

```diff
@@ -41,11 +41,11 @@ begin
       AND tablename   = 'log$_' || source_table
     ))
     THEN
-      EXECUTE 'DROP TRIGGER IF EXISTS ' || source_table || '_trigger' || ' ON ' || source_table;
+      EXECUTE 'ALTER TABLE ' || source_table || ' DISABLE TRIGGER ' || source_table || '_trigger';
```

Comment:
There doesn't seem to be a concise way of checking whether a trigger exists before attempting to disable/enable it. Fortunately, all of the source tables have the trigger on them, but this would be an issue if we added any source tables that didn't in future.

Reply:
Ah that's annoying... seems like it throws an error when attempting to disable a non-existent trigger... I guess we can just cross this bridge if and when we come to it 👍